README

```diff
@@ -0,0 +1,37 @@
+loadflex: Models and Tools for Watershed Flux Estimates
```

Comment:
ok, I just noticed that I had a file called README that wasn't in the repository and I assumed it was removed by mistake. But that sounds good Ill remove it again.